libs/ngx-dsfr/stepper/src/lib/stepper.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="fr-stepper">
+  <h2 class="fr-stepper__title">
+    <span class="fr-stepper__state">{{stepSubtitle}}</span>
+    {{stepTitle}}
+  </h2>
+  <div class="fr-stepper__steps" [attr.data-fr-current-step]="stepNumber" [attr.data-fr-steps]="numberOfSteps"></div>
+  <p class="fr-stepper__details" *ngIf="stepNumber !== numberOfSteps">
+    <span class="fr-text--bold">{{nextStepSubTitle}}</span> {{nextStepTitle}}
```

Comment:
Same thing here with the `nextStepSubTitle` property. You can replace this with : 
```html
<span class="fr-text--bold">Étape suivante :</span> {{nextStepTitle}}
```